MANIFEST.in

```diff
@@ -13,4 +13,6 @@ include HACKING.txt hacking-tox.ini
 include builddocs.sh coverage.sh scaffoldtests.sh
 include TODO.txt
 
+include .github .github/*
```

Comment:
graft

Reply:
I don't understand "graft". Sorry.

I read https://pypi.org/project/check-manifest/ and got this:

```
missing from sdist:
  .github
  .github/ISSUE_TEMPLATE
  .github/ISSUE_TEMPLATE/bug_report.md
  .github/ISSUE_TEMPLATE/documentation.md
  .github/ISSUE_TEMPLATE/feature_request.md
don't know how to come up with rules matching any of the files, sorry!
```

So I tried that locally as well, it failed but for different reasons, but I committed and pushed. We'll see if this guess is right. I have no clue what I'm doing.

Reply:
https://docs.python.org/2/distutils/sourcedist.html#commands read the docs and run `tox -e lint` locally if you aren't sure before you push.

Reply:
`tox -e lint` always fails for me locally with a bajillion `*.DStore`s, so I don't pay it any mind:

```
missing from VCS:
lists of files in version control and sdist do not match!
missing from VCS:
  docs/.DS_Store
  docs/_templates
  docs/_templates/.DS_Store
...
```

Reply:
OK, I think this is good now. Thanks for the link to the docs. Had to play around a bit to see what fails locally vs. in Travis.